templates/web/base/reports/_list-filters.html

```diff
@@ -9,9 +9,21 @@
         <option value="shortlisted"[% ' selected' IF filter_status.shortlisted %]>[% loc('Shortlisted') %]</option>
         <option value="unshortlisted"[% ' selected' IF filter_status.unshortlisted %]>[% loc('Unshortlisted') %]</option>
       [% END %]
+      [% IF c.user_exists AND c.user.is_superuser OR c.user.belongs_to_body(body.id) %]
+        <option value="open"[% ' selected' IF filter_status.open %]>[% loc('New reports') %]</option>
```

Comment:
I guess they're hoping they're new, but if they get reopened by a user etc, they might not be? Looks like Oxon have "New" group covering confirmed and investigating. Perhaps "Open" to match that?
![screen shot 2017-08-15 at 14 21 18](https://user-images.githubusercontent.com/154364/29317693-0d2c7256-81c5-11e7-98ba-e2d25872e73d.png)